testsuite/MDAnalysisTests/analysis/test_hole2.py

```diff
@@ -98,6 +98,23 @@ def test_symlink_file(self, tmpdir):
             assert not fixed.endswith(long_name)
 
 
+class TestOserror(object):
+    u = mda.Universe(MULTIPDB_HOLE)
```

Comment:
You should use a fixture here rather than defining the universe this way, see lines 238 to 240.